docs/source/releases/v1_10_0_dev0.rst

```diff
@@ -1,12 +1,41 @@
 Version 1.10.0.dev0 (2023-04-04)
 ********************************
 
+* Add YAML-based plugin validator
```

Comment:
Note I added bulleted summary list right under "Version" header, and made the "From issue" lines bold rather than a bullet.

Reply:
Got it. I will try to do this from now on.

Reply:
Okay, understood.